packages/eslint-plugin-next/src/rules/no-browser-api-server-component.ts

```diff
@@ -0,0 +1,96 @@
+import { defineRule } from '../utils/define-rule'
+import { isCapitalized } from '../utils/regex'
+
+const description = 'Prevent the use of browser APIs in server components.'
+
+const url =
+  'https://nextjs.org/docs/messages/no-browser-api-in-server-component'
+
+const browserAPIs = [
```

Comment:
```suggestion
const browserAPIs = new Set([
```
Using `Set` is better for performance